javascript/src/test/components/ChannelMessages/MessageScrollable.test.js

```diff
@@ -112,4 +112,26 @@ describe("MessageScrollableView tests", () => {
         }, 500)
     });
 
+    test("Links are clickable", () => {
+        useSWR.mockReturnValue({
+            data: []
+        });
+        const exampleMessage = {
+            "type": "message",
+            "subtype": "channel_join",
+            "ts": "1594143066.000200",
+            "user": "U017218J9B3",
+            "text": "Office hours at https://ucsb.zoom.us/j/89220034995?pwd=VTlHNXJpTVgrSEs5QUtlMDdqMC9wQT09",
```

Comment:
I just fixed the test, thank you for the feedback!
As far as the fake links, the problem is that if I specify a regex for http, then anything else within <>, such as channel notifications, disappear completely from the message text. I think this has to do with the fact that dangerouslySetInnerHTML causes React to interpret the entirety of the text as if it were HTML, which causes <> to be an HTML element. I thought it would be better to display those values as links that don't actually go anywhere rather than have them disappear completely. Do you think there's a better workaround, or is it ok to have those elements disappear?

Reply:
I see, that makes sense - well it's better to not even have them be as links, and just be text. 

Then I think a good workaround is changing the replaceLink function to a new function(need to rename to something that describes this new function better) that just finds all strings like this< text > and check the text string inside. If the text is a link format then actually replace it with a link like you already have, but if its not a link format, just remove the brackets and replace with just the text by itself so it doesn't try to render as html. I would also put a comment that this will leave the other cases like the @channel or emails/telephones looking weird make a new issue for those cases as future work.

Also going back a case I'm seeing is that if something is formatted as a link (rather than just a pasted direct link) it will be formatted like this:
<text the link is formatted to|https://dev.to/liviufromendtest/how-to-test-emails-with-selenium-5c5m>, this is a common case I think you should account for. This is the case described in Conrad's Issue request.

Reply:
I just made these changes! Please let me know if there's anything else I should fix.